index.css

```diff
@@ -521,21 +522,13 @@ section ul, section li {
     }
     
     h4 a {
-        font-size: 18px;
-        word-wrap: break-all;
+        word-break: break-all;
```

Comment:
BTW, I believe I (mostly) worked around this issue in my original design by giving `word-break: break-all` only to the `<code>` elements. Restating my request to keep the `` ` ``s in titles :smile: